
Header header

# Set either node_id or node_label
int32 node_id
string node_label